guides/common/attributes-base.adoc

```diff
@@ -106,9 +106,10 @@
 :postgresql-conf-dir: {postgresql-data-dir}
 :postgresql-log-dir: {postgresql-data-dir}/log
 :PIV: PIV
-:project-client-name: https://yum.theforeman.org/client/{ProjectVersion}/
-:project-client-RHEL7-url: https://yum.theforeman.org/client/{ProjectVersion}/el7/x86_64/foreman-client-release.rpm
-:project-client-RHEL8-url: https://yum.theforeman.org/client/{ProjectVersion}/el8/x86_64/foreman-client-release.rpm
+:project-client-url: https://yum.theforeman.org/client/{ProjectVersion}
+:project-client-name: {project-client-url}[Foreman Client]
```

Comment:
I always thought it was confusing that `project-client-name` holds the URL in upstream where in Satellite it's actually a name. Adding a label looks like a decent workaround.